### DISCUSSION TITLE
Expand recently added section

### DISCUSSION THREAD

**Discussion by PleaseStopAsking** [LeoKlaus/plappa#10] [Feature Requests]
The recently added section on the home screen appears to only be showing a small number of audiobooks currently. It does not appear to be restricted to something such as the last 30 days that I can tell but It would be nice if it included maybe the last 25 or at least more than what fits on the screen.

Somewhat related, it would also be nice to have the ability to open a new view for each section on the home page with a VStack list similar to the `Books` section but filtered to the specific section items.

**Comment by LeoKlaus** [comment]
I absolutely agree.
The current implementation was made with the list view in mind instead of the grid, so displaying tens of recently added books would move the sections below down too much. That's why I chose to cap recent items at 3.

But with the grid and an option to get to the full list, I don't see why there should be only 3 items in each section.

**Comment by LeoKlaus** [comment]
Build 20 adds the ability to view each section as list and increased the amount of items shown to 8 for each row in the grid view.
